backend/src/org/commcare/resources/ResourceManager.java

```diff
@@ -232,7 +232,7 @@ public void upgrade()
                 //everything here? Locale files are registered elsewhere, and we
                 //can't guarantee we're the only thing in there, so we can't
                 //straight up clear it...
-                platform.initialize(masterTable);
+                platform.initialize(masterTable, true);
```

Comment:
Ah, good call. I added a comment in the code warning for this and we can follow up in another PR to fix it.